src/server/src/main/java/io/cassandrareaper/service/RepairRunner.java

```diff
@@ -149,6 +149,11 @@ static int getPossibleParallelRepairsCount(
       throw new ReaperException(msg);
     }
 
+    LOG.debug(
+        "Possible parallel repairs : {}",
+        Math.min(
+            ranges.size() / ranges.values().iterator().next().size(),
+            Math.max(1, hostsInRing.keySet().size() / ranges.values().iterator().next().size())));
```

Comment:
maybe wrap this debug statement inside `if (LOG.isDebugEnabled()) {…}`

Reply:
Not really concerned with performance here, I'd rather avoid adding lines and conditional blocks if you don't mind.